pkg/cmd/cli/describe/printer.go

```diff
@@ -957,8 +957,13 @@ func printClusterNetworkList(list *sdnapi.ClusterNetworkList, w io.Writer, opts
 }
 
 func printEgressNetworkPolicy(n *sdnapi.EgressNetworkPolicy, w io.Writer, opts kctl.PrintOptions) error {
-	_, err := fmt.Fprintf(w, "%s\n", n.Name)
-	return err
+	if _, err := fmt.Fprintf(w, "%s\t", n.Namespace); err != nil {
+		return err
+	}
+	if _, err := fmt.Fprintf(w, "%s\n", n.Name); err != nil {
```

Comment:
Hm... somehow I thought this matched the style elsewhere, but I guess not really.
Namespace gets added automatically, although it looks like I didn't handle it quite correctly.

(Also, this was a fix-up that I squashed into the wrong commit, it should have gone into the earlier commit...)